test/browser/index.js

```diff
@@ -3,12 +3,14 @@ import sinonChai from 'sinon-chai';
 
 chai.use(sinonChai);
 
-import 'babel-polyfill';
+import 'core-js';
+import 'regenerator-runtime';
```

Comment:
Why do we need this btw? Surely all the browsers we need to support have async/await natively

Reply:
But the tests in this repo already do use async/await, so what changed here that you need regenerator all of a sudden now?

Reply:
`babel-polyfill` had sth similar to `regenerator-runtime` built in.. but it has wrong behavior when it comes to `JSON.stringify(new Map())`

Reply:
give https://github.com/mixpanel/panel/pull/153 a try :trollface:

Reply:
but you still use regenerator-runtime in that branch...

Reply:
ah I recall, so still have some older browser tests. i could just bump browser version and specify browserlist correctly

Reply:
yeah, that's what i mean about targeting a more modern environment